src/services/pipewire/defaults.cpp

```diff
@@ -146,7 +146,7 @@ void PwDefaultTracker::onNodeDestroyed(QObject* node) {
 
 void PwDefaultTracker::changeConfiguredSink(PwNode* node) {
 	if (node != nullptr) {
-		if (!node->isSink) {
+		if (node->type != PwNodeType::AudioSink) {
```

Comment:
This prevents an application from being set as the default due to apps having the Stream flag. Instead use [`testFlags`](https://doc.qt.io/qt-6/qflags.html#testFlags). `node->type.testFlags(PwNodeType::AudioSink)` should do it.